xml/System.ComponentModel.Design/DesignerActionService.xml

```diff
@@ -603,11 +603,26 @@
         <Parameter Name="type" Type="System.Windows.Forms.Design.ComponentActionsType" Index="1" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="component">To be added.</param>
-        <param name="type">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="component">The component that the smart tags are associated with.</param>
```

Comment:
IIRC @merriemcgaw mentioned we weren't allowed to use "smart tag" due to some litigation that we lost. I think we ought to use "designer action" instead.

Reply:
Yep, let's replace with DesignerActions wherever we come across it.